lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -15,20 +15,85 @@
  */
 'use strict';
 
-/* globals self */
+/* globals self Util */
+
+/**
+ * Create render context for critical-request-chain tree display.
+ * @param {!DetailsRenderer.CRCNode} tree
+ * @return {!{tree: DetailsRenderer.CRCNode, startTime: number, transferSize: number}}
+ */
+function initTree(tree) {
```

Comment:
these methods are called `createRenderTreeContext` and `createContextFor` in handelbar-helpers.